### PR TITLE
Solved minor Multilanguage support issues

### PR DESCRIPTION
**Pull request by morloi**
This is just a bunch of names and strings without multilanguage support (mainly integration descriptions and list details columns names) or with wrong text-domain.

### CODE REVIEW

File: includes/integrations/views/integration-settings.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 			</div>
 
 			<p>
-				<?php echo $integration->description; ?>
+				<?php _e($integration->description, 'newsletter-for-wp'); ?>
```
Comment (dannyvankooten) [inline includes/integrations/views/integration-settings.php]: Hi @morloi, could you change this to `mailchimp-for-wp`?

**Comment by dannyvankooten** [comment]
Hey @morloi, great catches, thank you for submitting a pull request!

File: includes/integrations/views/integration-settings.php (hunk 1 above)
Comment (morloi) [reply includes/integrations/views/integration-settings.php]: I've corrected these ones ;)

**Comment by dannyvankooten** [comment]
Awesome, thank you so much @morloi!

**Pull request merged**